Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -526,30 +661,24 @@ - (NSString*)descriptionWithLocale:(id)locale indent:(NSUInteger)level {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (id)copyWithZone:(NSZone*)zone {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self retain];
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (NSUInteger)countByEnumeratingWithState:(NSFastEnumerationState*)state objects:(id _Nonnull[])stackbuf count:(NSUInteger)len {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [[self array] countByEnumeratingWithState:state objects:stackbuf count:len];
```

Comment:
Is it legal to call this on a different NSArray object every time we're invoked? Can the state parameter be migrated from one object to another? I don't know the answer.